templates/search/search.html

```diff
@@ -77,6 +105,18 @@
         	}
         }
 
+        // Ajax request clustering
+        {% if clustering_on %}
```

Comment:
I'm not sure about these types of template tags directly in javascript. I would prefer:
```js
const clustering_on = {{clustering_on}};
...
if (clustering_on) {
   ...
}
```

Reply:
Then I need to do something like this:

```js
const clustering_on = "{{clustering_on}}";
...
if (clustering_on == "True") {
    ...
}
```

Which does not look very good.

Reply:
you could make the value of `{{clustering_on}}` a string that looks like a javascript boolean. So the final output javascript will look like
```js
const clustering_on = true;
```